### PR TITLE
BD-2695 Add image specs to an includes file

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #BD-2695

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

### CODE REVIEW

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Hey @rachel-feinberg! Thanks for tackling this! The includes usage looks great and is rendering correctly for me, however, I don't think this is the right place to put this content. This would mean we have all the channels image specs hosted under IAM modals. 😅  

I took a look at the ticket request and if I had to guess, it looks like they put a random URL that currently has image specs as the request URL, since that field is required. 

I'd suggest we do this instead:

1. Create a new file under [Templates and Media](https://braze-docs-ghmxy6lo3-braze.vercel.app/docs/user_guide/engagement_tools/templates_and_media)
2. Add all the content you just added (still use the includes!)
3. As a next step, find areas in the docs where we list image specs and replace them with the respective includes content.

Let me know what you think! Happy to chat it through in Slack too.

File: _docs/_user_guide/engagement_tools/templates_and_media/image_specs.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+---
+
+# Image specifications
+
+> In general, smaller and high-quality images will load faster, so we recommend using the smallest asset possible to achieve your desired output. To maximize your image use in specific channels, refer to the details in this article.
+
+You should always [preview and test your messages]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/testing/) on a variety of devices to confirm that the most important areas of your image and message appear as expected.
+
+{% alert tip %} Create assets with confidence! Our in-app message image templates and safe zone overlays are designed to play nicely with devices of all sizes. [Download Design Templates ZIP]({% image_buster /assets/download_file/Braze-In-App-Message-Design-Templates.zip %}) {% endalert %}
```
Comment (internetisaiah) [inline _docs/_user_guide/engagement_tools/templates_and_media/image_specs.md:20]:
```suggestion
{% alert tip %} Create assets with confidence! Our in-app message image templates and safe-zone overlays are designed to play nicely with devices of all sizes. [Download Design Templates ZIP]({% image_buster /assets/download_file/Braze-In-App-Message-Design-Templates.zip %}). {% endalert %}
```

File: _includes/image_specs.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    <td>PNG, JPG</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Android expanded notification image</td>
+    <td>2:1</td>
+    <td>Small: 512 x 256 px<br>Medium: 1024 x 512 px<br>Large: 2048 x 1024 px</td>
+    <td>500 KB</td>
+    <td>PNG, JPG</td>
+    <td>Used in <a href="{{site.baseurl}}/user_guide/message_building_by_channel/push/android/rich_notifications/">Android rich notifications</a></td>
```
Comment (internetisaiah) [inline _includes/image_specs.md:97]:
```suggestion
    <td>Used in <a href="{{site.baseurl}}/user_guide/message_building_by_channel/push/android/rich_notifications/">Android rich notifications</a>.</td>
```

**Review by internetisaiah — COMMENTED** [review]
Just tiny suggestions. Docs look great!

**Review by internetisaiah — APPROVED** [review]
LGTM

**Pull request merged**